Update contributors page (#1140)
demisto/content-docs: docs/packs/packs-format.md

 
In case file does not exist, the pack author name will be displayed in text.

### CONTRIBUTORS.json

> If you are contributing to an existing pack, you can add a **CONTRIBUTORS.json** file to the root of the pack in case one does not already exist. The file should contain a list of strings that will include your name. 

Reviewer: ```suggestion
If you are contributing to an existing pack, you can add a **CONTRIBUTORS.json** file to the root of the pack in the event that one does not already exist. The file should contain a list of strings that will include your name. 
```

> If you are contributing to an existing pack, you can add a **CONTRIBUTORS.json** file to the root of the pack in case one does not already exist. The file should contain a list of strings that will include your name. 

Reviewer: ```suggestion
If you are contributing to an existing pack, you can add a **CONTRIBUTORS.json** file to the root of the pack in case one does not already exist. The file should contain a list of strings including your name. 
```

#### Example of a CONTRIBUTORS.json file:
```json
[
    "Jane Doe",
    "John Smith"
]
```

> #### Once your contribution will be merged, pack details will show the following:

Reviewer: ```suggestion
#### Once your contribution is merged, pack details will show the following:
```
